frontend/udp/bytepool/bytepool.go

```diff
@@ -11,24 +11,27 @@ type BytePool struct {
 func New(length int) *BytePool {
 	var bp BytePool
 	bp.Pool.New = func() interface{} {
-		return make([]byte, length)
+		b := make([]byte, length)
+		return &b
```

Comment:
Huh. I went down the rabbit hole, which also lead me [here](https://github.com/mailru/easyjson/issues/157).

If I understand this correctly, the comment on staticcheck.io is a bit misleading. While it's true that "passing a value that is not a pointer to a function that accepts an interface, the value needs to be placed on the heap", I did not immediately see how this affects us, as all the network functions we use take byte slices, not interfaces. From above github discussion, however, I'm now under the impression that `(*sync.Pool) Put(x interface{})` itself is the culprit...

Very weird. But nice catch!